lib/ImportStatement.js

```diff
@@ -37,6 +38,7 @@ export default class ImportStatement {
   originalImportString: ?string;
   leadingContent: ?string;
   path: string;
+  config: Configuration;
```

Comment:
Instead of passing in the whole config object, could you pass in only the new option? That would make the code a bit more explicit, and it's more in line with what we're doing elsewhere.